src/App.js

```diff
@@ -5,18 +5,32 @@ import Nav from './components/Nav';
 import DisplayArticles from './components/DisplayArticles';
 import ArticleCard from './components/ArticleCard';
 import DisplayComments from './components/DisplayComments';
+import CheckConnection from './components/CheckConnection';
+import Login from './components/Login';
+import LoginPage from './components/LoginPage';
+import AddCommentForm from './components/AddCommentForm';
+import { useEffect, useState } from 'react';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState({})
+  const [isUserLoggedIn, setIsUserLoggedIn] = useState(false)
 
+  useEffect(() => {
+
+  }, [currentUser, isUserLoggedIn])
```

Comment:
this useEffect appears to be empty, if you don't need it I would recommend removing